myuw/templates/handlebars/card/schedule/visual.html

```diff
@@ -4,29 +4,63 @@
 
 <div class="card"  data-name="VisualScheduleCard" data-type="card">
     {{#if total_sections}}
-   <!-- {{# if has_early_fall_start }}
-    <a href="#" class="show_full_term_meetings">Show {{ucfirst quarter}} Full Term Courses</a>
-    <a href="#" class="show_efs_meetings" style="display: none;">Show {{ucfirst quarter}} Early Fall Start Courses</a>
-    {{/if}}-->
-
     <h3 id="quarter-info" >
-            {{ucfirst quarter}} {{year}} Schedule
-            {{#if summer_term}}
-            <span class="shortTitle">{{summer_term}}</span>
-            {{/if}}
+        {{ titleFormatTerm term_string }} Schedule
+
+    </h3>
+            <span class="sr-only">Final exam schedule and Visual Schedule</span>
     </h3>
```

Comment:
Extra `</h3>` tag here. 
Also, the sr-only text is pretty unnecessary. Above it already has a title for the "Term Schedule", and then below there is an H4 header that says Final Exam Schedule.